frontend/src/PrivateRoute.tsx

```diff
@@ -21,7 +21,11 @@ const PrivateRoute = (props: IPrivateRouteProps) => {
       {...rest}
       render={props => {
         if (!!keycloak.obj?.authenticated) {
-          if (keycloak.hasRole(rest.role) || keycloak.hasClaim(rest.claim)) {
+          if (
```

Comment:
This will allow us to use PrivateRoutes for authenticated users without any special role/claim.